scripts/dotnet-cli-build/Utils/HostVersion.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class HostVersion : Version
+    {
+        // ------------------------------------------HOST-VERSIONING-------------------------------------------
+        //
+        // Host versions are independent of CLI versions. Moreover, these version numbers
+        // are baked into the binary and is used to look up a serviced binary replacement.
+        //
+
+        //
+        // Latest hosts for production of nupkgs.
+        //
+
+        // Version constants without suffix
+        public override int Major => 1;
```

Comment:
Instead of an override, would it make more sense to just set these properties in the constructor?